src/main/plugins/org.dita.pdf2/xsl/fo/index.xsl

```diff
@@ -256,7 +256,7 @@ See the accompanying LICENSE file for applicable license.
                         <xsl:with-param name="id" select="'Index See String'"/>
                     </xsl:call-template>
                 </fo:inline>
-                <xsl:for-each select="opentopic-index:index.entry">
+                <xsl:for-each select="opentopic-index:index.entry[opentopic-index:get-see-destination(.)]">
```

Comment:
If there are no valid See destinations, does that mean we will end up with an empty _See_ in the result? Same for below with _See also_.

That may be OK, because this is basically an error condition, but it might be good to put an error in the log that there were see/see-also entries but none were valid? Or a more general "Dropping see/see-also from X to Y because Y is not defined"?

Reply:
The validation should really be done in the index processing code, not this late in the chain. This is a quick fix to allow content to be processed in FOP, because FOP will crash on this type of bug.

Reply:
The validation should really be done in the index processing code, not this late in the chain. This is a quick fix to allow content to be processed in FOP, because FOP will crash on this type of bug.

Reply:
OK - works for me, and agree it should be done earlier.

Reply:
> it might be good to put an error in the log that there were see/see-also entries but none were valid

We have `DOTJ050W`:

> Found an `<index-see>` or `<index-see-also>` reference to the term '%1', but that term is not defined in the index.